activestorage/test/service/azure_storage_service_test.rb

```diff
@@ -29,7 +29,7 @@ class ActiveStorage::Service::AzureStorageServiceTest < ActiveSupport::TestCase
 
       @service.upload(key, StringIO.new(data), checksum: Digest::MD5.base64digest(data), filename: ActiveStorage::Filename.new("test.txt"), disposition: :inline)
 
-      assert_equal("inline; filename=\"test.txt\"; filename*=UTF-8''test.txt", @service.blobs.get_blob_properties(@service.container, key).properties[:content_disposition])
+      assert_equal("inline; filename=\"test.txt\"; filename*=UTF-8''test.txt", @service.get_blob_properties(@service.container, key).properties[:content_disposition])
```

Comment:
Shouldn’t this be `@service.client.get_blob_properties`?

```
Error:
ActiveStorage::Service::AzureStorageServiceTest#test_upload_with_content_disposition:
NoMethodError: undefined method `get_blob_properties' for #<ActiveStorage::Service::AzureStorageService:0x00007fa242abd830>
```

Reply:
You're right. Somehow I didn't catch this error when running the tests. My bad, sorry about that!